src/components/PurchaseSummary/PurchaseSummaryAccordion/style.tsx

```diff
@@ -0,0 +1,75 @@
+import styled from 'styled-components'
+import { Colors, mediaQueries } from '../../../tokens'
+
+export const AccordionContainer = styled.div`

```

Comment:
All styled components come with the Styled suffix. (AccordionContainerStyled, AccordionHeaderStyled, AccordionContentStyled...)
This way it is easier to differentiate common components from styled components.